ConnectorTeklaStructures/ConnectorTeklaStructures2022/packages.config

```diff
@@ -0,0 +1,51 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
```

Comment:
I don't think `packages.config` are used since we have SDK style projects and they can be removed

Reply:
I think this one is in use bc the 2022 project may have been generated from a template.

It is easy to upgrade to `PackageReferences` so I'd do that instead.

Reply:
Yeah, it's already using PackageReference ;)